apps/condo/domains/organization/schema/ResetOrganizationService.test.js

```diff
@@ -178,4 +201,37 @@ describe('ResetOrganizationService', () => {
 
     })
 
+    test('notification not send to resident after reset organization', async () => {
+        const client = await makeClientWithServiceConsumer()
+        const { property, organization, serviceConsumer, resident } = client
+        const [resource] = await MeterResource.getAll(admin, {})
+
+        const [meter] = await createTestMeter(admin, organization, property, resource, {
+            accountNumber: serviceConsumer.accountNumber,
+            unitName: resident.unitName,
+            verificationDate: dayjs().add(-1, 'year').toISOString(),
+            nextVerificationDate: undefined,
+        })
+
+        await createTestMeterReportingPeriod(admin, organization, {
+            notifyStartDay: 1,
+            notifyEndDay: Number(dayjs().format('DD')),
+        })
+
+        await resetOrganizationByTestClient(admin, { organizationId: organization.id })
+
+        await sendSubmitMeterReadingsPushNotifications()
+
+        await waitFor(async () => {
+            const messages = await Message.getAll(admin, {
+                user: { id: client.user.id },
+                type_in: [
+                    METER_SUBMIT_READINGS_REMINDER_END_PERIOD_TYPE,
+                ],
+            })
+            
+            messages.filter(message => message.meta.data.meterId === meter.id)
+            expect(messages).toHaveLength(0)
+        })
```

Comment:
If in parallel in another test before the organization reset the sending of notifications starts, then your test will not pass, because the message will be sent
This is a global problem, but I do not yet understand how it can be overcome

Reply:
Wouldn't a message type filter help in this case?

Reply:
> Wouldn't a message type filter help in this case?

No, same push notifications are being sent out